Amperes-Selam-Ainalem-Solar-System.rb

```diff
@@ -0,0 +1,118 @@
+class SolarSystem
+  # initialized SolarSystem object with an array of Planet objects.
+  attr_reader:solar_sys_age
+  def initialize(user_planet,solar_sys_age)
+    @planets = user_planet
+    @solar_sys_age = solar_sys_age
+  end
+  # adds a Planet object to an array of Planets
+  def add_planet(new_planet)
+    @planets.push(new_planet)
+    return @planets
+  end
+  # iterates through array of Planet objects and displays a formatted list to user.
+  def list_of_planets()
+    counter = 0
+    planet_names = ""
+    @planets.each do |planet|
+      planet_names += "\n#{counter+1}. #{planet.planet_details}"
+      counter+=1
+    end
+    return planet_names
+  end
+
+end
+
+
+
+class Planet
+  attr_reader:name, :radius_mi, :distance_frm_sun, :num_moons, :num_suns, :year_in_days
```

Comment:
A small thing, but important. This syntax is wrong because you need a space between `attr_reader` and `:name` so it should be:
```ruby
attr_reader:name, :radius_mi, :distance_frm_sun, :num_moons, :num_suns, :year_in_days
```